stories/cosmoz-input.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
what's up with this file ?

Reply:
Personal VS Code settings which went in the PR by mistake. Reverting.